file/test.cc

```diff
@@ -49,6 +49,24 @@ TEST(File, JoinPath) {
   ASSERT_THROW(FileSystem::JoinPath("/foo/", ""), FileException);
 }
 
+TEST(File, GetFileExtension) {
+  EXPECT_EQ("", FileSystem::GetFileExtension(""));
```

Comment:
No. If a test fails, the user should immediately see which one did, by the line number.

(This is one of many small things I leaned in Google.)